cpp/include/graph.hpp

```diff
@@ -257,9 +266,12 @@ class GraphCSRView : public GraphCompressedSparseBaseView<VT, ET, WT> {
    * @param  number_of_vertices    The number of vertices in the graph
    * @param  number_of_edges       The number of edges in the graph
    */
-  GraphCSRView(
-    ET *offsets_, VT *indices_, WT *edge_data_, VT number_of_vertices_, ET number_of_edges_)
-    : GraphCompressedSparseBaseView<VT, ET, WT>(
+  GraphCSRView(edge_t *offsets_,
+               vertex_t *indices_,
+               weight_t *edge_data_,
+               vertex_t number_of_vertices_,
+               edge_t number_of_edges_)
```

Comment:
variable names with trailing "_" is for class private member variables (RAFT clang-tidy rules).

Reply:
fixed